status-go-version.json

```diff
@@ -3,7 +3,7 @@
     "_comment": "Instead use: scripts/update-status-go.sh <rev>",
     "owner": "status-im",
     "repo": "status-go",
-    "version": "v0.172.8",
-    "commit-sha1": "436d22985661322ffda4eb44c1a160e6804f3823",
-    "src-sha256": "1y3l469k2085qaml7dmaky1rlanl4phq2p6yyk97074yw839jzj1"
+    "version": "feat/update-display-name",
```

Comment:
@mohsen-ghafouri, this branch implements an endpoint that's very similar to the existing `SetDisplayName` in status-go. I saw someone else commenting there the same. Anyway, the rest of this PR looks good :)

Reply:
yes had a conversation with Andrea, and i reverted changes and just fix the issue in status-go